php_phongo.c

```diff
@@ -136,10 +136,16 @@ zend_class_entry* phongo_exception_from_phongo_domain(php_phongo_error_domain_t
 	MONGOC_ERROR("Resolving unknown phongo error domain: %d", domain);
 	return php_phongo_runtimeexception_ce;
 }
-zend_class_entry* phongo_exception_from_mongoc_domain(uint32_t /* mongoc_error_domain_t */ domain, uint32_t /* mongoc_error_code_t */ code)
+zend_class_entry* phongo_exception_from_mongoc_domain(mongoc_error_domain_t domain, mongoc_error_code_t code)
```

Comment:
The old signature dates back to 8710d74ccbdf59af261bd89e132c5a62db48bbd3, but I don't recall why `uint32_t` was used instead of the enums. Assuming this doesn't product any compiler warnings, it seems OK to me.

Reply:
The old logic used a `switch` statement to check the error codes, which can cause compiler warnings if not all values from `mongoc_error_domain_t` are covered. The code compiles without warning, so I assume that the reason for using `uint32_t` has gone away.